irohad/model/permissions.hpp

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright Soramitsu Co., Ltd. 2016 All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#ifndef IROHA_PERMISSIONS_HPP
+#define IROHA_PERMISSIONS_HPP
+
+#include <string>
+
+namespace iroha {
+  namespace model {
+
+  const std::string can_append_role = "CanAppendRole";
```

Comment:
Okay, this might be noisy but this is true. We need to put consolidated information about current permissions and their mapping to docs. How about a separate page for permissions on iroha-api website?

Else, we need to put short comment for the permissions into the code directly. 
If we do both — this would be nice for developers anyway.